src/normalize.js

```diff
@@ -105,6 +105,8 @@ function extractToVirtualKey(uri, rel, content, opts) {
     [opts.metaKey]: {
       self: virtualKey,
       virtual: true,
+      parentResource: uri,
+      parentProperty: rel,
```

Comment:
In HAL, I think the proper term would be `rel` or `relation` instead of `property`